src/App.tsx

```diff
@@ -1,14 +1,39 @@
 import './index.css'
 import { FirstButton } from './components/Button'
+import { useState } from 'react'
+import { BiGame, BiRocket, BiSmile } from 'react-icons/bi'
+
 
 function App() {
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  function handleClick() {
```

Comment:
any parts of this that are required for the component must live in the component itself, not in App.tsx